tests/Aggregate/AggregateTranslatorConfigurationTest.php

```diff
@@ -0,0 +1,327 @@
+<?php
+/**
+ * This file is part of the prooph/service-bus.
+ * (c) 2014-2016 prooph software GmbH <contact@prooph.de>
+ * (c) 2015-2016 Sascha-Oliver Prolic <saschaprolic@googlemail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace ProophTest\EventStore\Aggregate;
+
+use PHPUnit_Framework_TestCase as TestCase;
+use Prooph\EventStore\Aggregate\AggregateTranslatorConfiguration;
+use Prooph\EventStore\Aggregate\AggregateType;
+use Prooph\EventStore\Aggregate\AggregateTypeProvider;
+use ProophTest\EventStore\Mock\Post;
+use ProophTest\EventStore\Mock\User;
+
+/**
+ * Class AggregateTypeTest
+ *
+ * @package ProophTest\EventStore\Aggregate
+ */
+class AggregateTranslatorConfigurationTest extends TestCase
+{
+    /**
+     * @var AggregateTranslatorConfiguration
+     */
+    private $config;
+
+    protected function setUp()
+    {
+        parent::setUp();
+        $this->config = AggregateTranslatorConfiguration::createWithDefaults();
+    }
+
+    /**
+     * @test
+     * @skipped
+     * @expectedException \InvalidArgumentException
```

Comment:
ah this is for master branch, not develop, sorry, my bad.

would you consider another PR for develop?